src/Components/Todo/Todo.tsx

```diff
@@ -0,0 +1,75 @@
+import { AuthContext } from '@/Context/AuthContext';
+import { ToastContext } from '@/Context/ToastContext';
+import baseUrl from '@/utils/Custom/Custom';
+import { useContext } from 'react';
+import { useDrop } from 'react-dnd';
+import { Task } from '..';
+import './Todo.module.scss';
+import { IEmpTasks } from '@/Interfaces/EmployeeTasks';
+
+
+
+
+
+const Todo = ({ AllTasks,getTasksData }: IEmpTasks) => {
```

Comment:
We have three components: ToDo, InProgress, and Done. They're quite similar, and updating them requires changes in three separate files. To simplify, we can create a single column component, serving as the main reference. This not only streamlines edits but also makes adapting to potential new statuses more straightforward, avoiding code complexity.